Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -187,7 +187,7 @@ public class SessionController : BaseJellyfinApiController
         /// <param name="playstateRequest">The <see cref="PlaystateRequest"/>.</param>
         /// <response code="204">Playstate command sent to session.</response>
         /// <returns>A <see cref="NoContentResult"/>.</returns>
-        [HttpPost("Sessions/{sessionId}/Playing")]
+        [HttpPost("Sessions/{sessionId}/Playing/{command}")]
         [Authorize(Policy = Policies.DefaultAuthorization)]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         public ActionResult SendPlaystateCommand(
```

Comment:
Add `[FromRoute, Required] PlayCommand command,` below sessionId and change

```
[FromBody] PlaystateRequest playstateRequest
```
to
```
[FromQuery] PlaystateRequest playstateRequest
```

Reply:
@cvium Have i understood you right? You mean like below? If so, command isn't used.

```
        [HttpPost("Sessions/{sessionId}/Playing/{command}")]
        [Authorize(Policy = Policies.DefaultAuthorization)]
        [ProducesResponseType(StatusCodes.Status204NoContent)]
        public ActionResult SendPlaystateCommand(
            [FromRoute, Required] string sessionId,
            [FromRoute, Required] PlayCommand command,
            [FromQuery] PlaystateRequest playstateRequest)
        {
            _sessionManager.SendPlaystateCommand(
                RequestHelpers.GetSession(_sessionManager, _authContext, Request).Id,
                sessionId,
                playstateRequest,
                CancellationToken.None);

            return NoContent();
        }
```

Reply:
Yes. That seems to be in line with the jf-web and the servicestack api.

Reply:
been back to v6 code:

It's defined as 

```
[Route("/Sessions/{Id}/Playing/{Command}", "POST", Summary = "Issues a playstate command to a client")]
    [Authenticated]
    public class SendPlaystateCommand : PlaystateRequest, IReturnVoid
```

with PlaystateRequest as:-

```
public class PlaystateRequest
    {
        public PlaystateCommand Command { get; set; }

        public long? SeekPositionTicks { get; set; }

        /// <summary>
        /// Gets or sets the controlling user identifier.
        /// </summary>
        /// <value>The controlling user identifier.</value>
        public string? ControllingUserId { get; set; }
    }
```
Base on this, wouldn't the correct definition be:-
```
public ActionResult SendPlaystateCommand(
            [FromRoute, Required] string sessionId,
            [FromRoute, Required] PlayCommand command,
            [FromQuery] long? seekPositionTicks,
            [FromQuery] string? controllingUserId)
        {
            var playstateRequest = new PlayStateRequest()
            {
                  Command = command,
                  ControllingUserId = controllingUserId,
                  SeekPositionTicks = seekPositionTicks,
            };

            _sessionManager.SendPlaystateCommand(
                RequestHelpers.GetSession(_sessionManager, _authContext, Request).Id,
                sessionId,
                playstateRequest,
                CancellationToken.None);

            return NoContent();
        }
```

Reply:
Yes, I suppose you're right